paymentsheet/src/test/java/com/stripe/android/customersheet/CustomerSheetScreenshotTest.kt

```diff
@@ -92,10 +92,10 @@ internal class CustomerSheetScreenshotTest {
         ),
         usBankAccountFormArguments = usBankAccountFormArguments,
         supportedPaymentMethods = listOf(
-            LpmRepository.HardcodedCard,
-            LpmRepository.hardCodedUsBankAccount,
+            LpmRepositoryTestHelpers.card,
+            LpmRepositoryTestHelpers.usBankAccount,
```

Comment:
❤️

Reply:
Having a shared factory for this is nice (and it's now test only!) But the pattern sucks!  😝